### PR TITLE
optionally add logits to model output

### PR DESCRIPTION
**Pull request by mwitiderrick**
```
from deepsparse import Pipeline
cv_pipeline = Pipeline.create(
  task='image_classification',
  model_path='zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none',  # Path to checkpoint or SparseZoo stub
  top_k=3,
  return_logits = True,
)
input_image = "cheetah.jpeg" # path to input image
inference = cv_pipeline(images=input_image)
inference

# ImageClassificationOutput(labels=None, scores=None, logits=array([[ 1.8277559 ,  4.486894  ,  2.4551017 ,  4.777242  ,  # 2.744733  ,.... 8.45131   ]],
 #  dtype=float32))
```

### CODE REVIEW

File: src/deepsparse/image_classification/schemas.py [hunk 1]
```diff
@@ -38,10 +38,15 @@ class ImageClassificationOutput(BaseModel):
...
+    class Config:
+        arbitrary_types_allowed = True
 
     labels: List[Union[int, str, List[int], List[str]]] = Field(
         description="List of labels, one for each prediction"
     )
     scores: List[Union[float, List[float]]] = Field(
         description="List of scores, one for each prediction"
     )
+    logits: Optional[ndarray] = Field(description="Raw model outputs")
```
Comment (rahul-tuli) [inline src/deepsparse/image_classification/schemas.py:52]: Could we test this logic with deepsparse server? I remember we had issues earlier when keeping numpy arrays in I/O schemas

File: src/deepsparse/image_classification/pipelines.py [hunk 2]
```diff
@@ -112,7 +114,7 @@ def class_names(self) -> Optional[Dict[str, str]]:
             mapping class ids to class labels
         """
         return self._class_names
-
+
```
Comment (dbogunowicz) [inline src/deepsparse/image_classification/pipelines.py:117]: Nit: uneccessary

File: src/deepsparse/image_classification/schemas.py (hunk 1 above)
Comment (dbogunowicz) [reply src/deepsparse/image_classification/schemas.py:52]: Yes, I second that!

Comment (dbogunowicz) [reply src/deepsparse/image_classification/schemas.py:52]: also good idea to set the default to None here

File: src/deepsparse/image_classification/schemas.py [hunk 3]
```diff
@@ -38,10 +38,15 @@ class ImageClassificationOutput(BaseModel):
     """
     Output model for image classification
     """
+    ndarray: numpy.ndarray = numpy.ndarray
```
Comment (dbogunowicz) [inline src/deepsparse/image_classification/schemas.py:41]: I'd say this adds unnecessary clutter. Let's just use for typing `numpy.ndarray`

File: src/deepsparse/image_classification/pipelines.py [hunk 4]
```diff
@@ -75,6 +75,7 @@ def __init__(
         *,
         class_names: Union[None, str, Dict[str, str]] = None,
         top_k: int = 1,
+        return_logits: bool = False,
```
Comment (bfineran) [inline src/deepsparse/image_classification/pipelines.py:78]: nit: update docstring as well

**Comment by mwitiderrick** [comment]
I am going to close this PR because I was able to get the logits through the Engine option.  Unless people think it's important to output the logits in the pipelines
